flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -356,8 +356,117 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
   // Math functions
   // ----------------------------------------------------------------------------------------------
 
+  @Test
+  def testAdd(): Unit = {
```

Comment:
Can you move this to `ScalarOperatorsTest`? These +/*/% are core operators.

Reply:
Make sense to me.